test/rg/injektor/FactoryOnlyDependencyInjectionContainerTest.php

```diff
@@ -51,14 +52,9 @@ private function processClassesOfFile(string $fileName, WritingFactoryGenerator
         require_once $fileName;
 
         $astLocator = (new BetterReflection())->astLocator();
-        // bc break between v4 and v5. Can be switched to DefaultReflector fully when min version is PHP 8.0
-        if (class_exists(\Roave\BetterReflection\Reflector\ClassReflector::class)) {
-            $reflector  = new \Roave\BetterReflection\Reflector\ClassReflector(new SingleFileSourceLocator($fileName, $astLocator));
-            $classes = $reflector->getAllClasses();
-        } else {
-            $reflector  = new \Roave\BetterReflection\Reflector\DefaultReflector(new SingleFileSourceLocator($fileName, $astLocator));
-            $classes = $reflector->reflectAllClasses();
-        }
+        $reflector  = new DefaultReflector(new SingleFileSourceLocator($fileName, $astLocator));
```

Comment:
After removing this, the composer.json needs to be adjusted to only allow `roave/better-reflection` 5.x but not 4.x.

Reply:
ah forget to remove it, thank you for pointing it out.